ocaml/idl/constants.ml

```diff
@@ -61,6 +61,7 @@ let message_put_uri = "/messages"                     (* ocaml/xapi/xapi_message
 let wlb_report_uri = "/wlb_report"                    (* ocaml/xapi/wlb_reports.ml *)
 let wlb_diagnostics_uri = "/wlb_diagnostics"          (* ocaml/xapi/wlb_reports.ml *)
 let audit_log_uri = "/audit_log"                      (* ocaml/xapi/audit.ml *)
+let get_pool_update_download_uri = "/update/"         (* ocaml/xapi/xapi_pool_update.ml *)
```

Comment:
Hi @lindig 
That's because I only want to serve for http://ip/update/*.
http://ip/update will not be a correct URL to access